### ISSUE TITLE
error message references OUTPUT_URL instead of BACKUP_URL

### ISSUE THREAD

**Issue by tcerna** [rear/rear#925]
* rear version (/usr/sbin/rear -V):
rear-1.17.2-1.el7.noarch

* OS version (cat /etc/rear/os.conf or lsb_release -a):
RedHatEnterpriseServer 7

* rear configuration files (cat /etc/rear/site.conf or cat /etc/rear/local.conf):
```
# cat /etc/rear/local.conf
BACKUP_URL=file://tmp/rearbackup/
```

* Brief description of the issue

    When running 'rear -v mkbackuponly', it raises this error:
```
ERROR: OUTPUT_URL requires tripple slash ...
```

* Steps to Reproduce:
   1. set BACKUP_URL into /etc/rear/local.conf and set path with only to slashes
   2. run rear -v mkbackuponly

* Actual results:
ERROR: OUTPUT_URL requires tripple slash ...

* Expected results:
ERROR: BACKUP_URL requires tripple slash ...

**Comment by jsmeix** [comment]
With https://github.com/rear/rear/pull/927
this issue should be fixed.

FYI:
Initially I thought it is only a typo
but actually it was more complicated
because of prep/default/02_translate_url.sh

**Comment by jsmeix** [comment]
I would very much appreciate it if I could get feedback
whether or not it now also works for you or
if I may have even introduced new issues.

In general regarding how to test the currently
newest rear GitHub master code:
<pre>
# git clone https://github.com/rear/rear.git

# cd rear

# vi etc/rear/local.conf

# usr/sbin/rear -d -D mkbackup
</pre>

**Comment by tcerna** [comment]
@jsmeix In which version of rear it is fixed?

**Comment by tcerna** [comment]
Sorry for spam, I found it in issue 927.
Fixed in Rear v1.19.

**Comment by jsmeix** [comment]
Strictly speaking "will be fixed in the upcoming version 1.19"
(provided you provide feedback that my https://github.com/rear/rear/pull/927
also makes it work for you).
